### DISCUSSION TITLE
How do I arrange mod load order

### DISCUSSION THREAD

**Discussion by mwpowellhtx** [Foxlider/FASTER#147] [Q&A]
Since because, at least last time I checked, _load order matters_, at least in some instances. i.e. generally wanting _CBA_A3_ before _ace_, or the RHS mods before their compats, along these lines. However, that's not the order in which they are presented, nor can I seem to _drag and drop_ at any of the pertinent direct or otherwise tangentially related mod lists, or otherwise _prioritize mod ordering_. Maybe the mod prioritization, tree shaking, pruning, etc, has improved? Unclear...

**Comment by MildlyInterested** [comment]
You don't need to worry about that. Arma itself will figure out the correct load order with the mods config. It might break if a mod is not configured properly, best case for that is to just not juse the mod. If you really want to edit it double click on the `LOAD` column and enter a number, load order is low to high.

**Reply by mwpowellhtx** [reply]
`LOAD` got it, thanks so much...

**Reply by OverlordZorn** [reply]
Wait... So small number means low priority?

**Reply by mwpowellhtx** [reply]
`LOAD` ordering, verified in the command line mapping. Which to clarify, lower numbers first.
